e2e/upgrades/upgrades_test.go

```diff
@@ -100,6 +100,59 @@ func TestUpgradesMinorVersion(t *testing.T) {
 	steps.Run(t)
 }
 
+// TestUpgradesMinorVersion tests a minor version bump
+func TestUpgradesMajorVersion20to21WithCockroachDbVersion(t *testing.T) {
```

Comment:
Can we add this to another pkg? The upgrade tests are taking time :(

Reply:
I would prefer to have identical test suites, one that runs the upgrade tests using image.name and one using cockroachDBversion so we can do this in parallel.  Thoughts?

Reply:
@keith-mcclellan I agree with you, because the same scenario must work also with api field

Reply:
@keith-mcclellan , @chrislovecnm , @udnay  I added in another package the tests with the related image in the sha256 format